main.go

```diff
@@ -62,13 +62,15 @@ func init() {
 
 func main() {
 	var enableLeaderElection bool
+	var disableLogging bool
```

Comment:
Why is that not enableLogging instead?

Reply:
Because with the current libs/way args are managed, it seems a boolean is set to true when the parameter is passed as argument.

So it started from the CLI arg, and I wanted to keep it consistent across the whole app.

Reply:
What do you prefer?

Reply:
I would then see another flag like enable-vintage (false by default) that would be true only if we are in vintage clusters

Reply:
I didn't manage to have a flag that is "true" by default and we can pass "=false" for it to be false.
I can do it with a string, but I thought that would be useless added complexity.

Reply:
That's weird because it works on my machine :D 

Sample code:

```
  1 
  2 package main
  3 
  4 // Go provides a `flag` package supporting basic
  5 // command-line flag parsing. We'll use this package to
  6 // implement our example command-line program.
  7 import (
  8         "flag"
  9         "fmt"
 10 )
 11 
 12 func main() {
 13         // It's also possible to declare an option that uses an
 14         // existing var declared elsewhere in the program.
 15         // Note that we need to pass in a pointer to the flag
 16         // declaration function.
 17         var bvar bool
 18         flag.BoolVar(&bvar, "bvar", true, "a bool var")
 19 
 20         // Once all flags are declared, call `flag.Parse()`
 21         // to execute the command-line parsing.
 22         flag.Parse()
 23 
 24         fmt.Println("bvar:", bvar)
 25 }
```


go run main.go
bvar: true
> go run main.go  --bvar=false
bvar: false
> go run main.go
bvar: true

Reply:
ok, yes it works if you do:
```
go run main.go --bvar=false
```

It won't if you do:
```
go run main.go --bvar false
```

I don't like that :disappointed: I think it's misleading.

Reply:
ok, it will be managed in the helm chart anyway, fine with me! :grin: 
I updated the code.